vcd/resource_vcd_nsxt_alb_service_engine_group.go

```diff
@@ -87,6 +87,13 @@ func resourceVcdAlbServiceEngineGroup() *schema.Resource {
 				Default:     false,
 				Description: "Boolean value that shows if sync should be performed on every refresh",
 			},
+			"supported_feature_set": {
+				Type:         schema.TypeString,
+				Optional:     true,
```

Comment:
Missing `Computed: true`

What happens now is if I don't set `supported_feature_set` (imagine I just migrated from <10.4) each apply will do this:
```
Terraform used the selected providers to generate the following execution plan. Resource actions are indicated with the following symbols:
  ~ update in-place

Terraform will perform the following actions:

  # vcd_nsxt_alb_service_engine_group.first will be updated in-place
  ~ resource "vcd_nsxt_alb_service_engine_group" "first" {
        id                                   = "urn:vcloud:serviceEngineGroup:b8a1b5b4-dbf0-45f5-8948-95e16a29479d"
        name                                 = "first-se"
      - supported_feature_set                = "STANDARD" -> null
        # (9 unchanged attributes hidden)
    }
```

While it does not happen for `vcd_nsxt_alb_settings` because it has `Computed: true`

Reply:
Good catch, totally true. I changed `vcd_nsxt_alb_settings` to `Computed: true` but forgot about this one.
Thanks!